src/components/AccountMenuDropdown.js

```diff
@@ -0,0 +1,34 @@
+import React, { Component } from 'react'
+import Menu from './Menu'
+import ReactCSSTransitionGroup from 'react-addons-css-transition-group'
+
+export default class AccountMenuDropdown extends Component {
+  constructor( props ) {
+
+    super( props )
```

Comment:
Lol, please remove whitespace above `super( props )`, and add it below. The goal is to separate setup of a function from the logic:
```js
constructor( props ) {
  super( props )

  this.state = { active: false }
}
```
Also, I just realized that `menuActive` is a little verbose - let's rename to `active` (what else would be active in this context, feels like there's no need to disambiguate).